Invalid or unknown abi 16 for function "_ZGVnN4vv_atan2f"

I'm trying to build https://github.com/tembo-io/pg_later on Ubuntu Noble with pg 16. The same setup works on Ubuntu Jammy with pg 16. This is what I ran:

```
RUST_BACKTRACE=1 cargo pgrx init --pg16 ...
RUST_BACKTRACE=1 cargo pgrx package --pg-config ... --out-dir ...
```

Error trace:

```
64.78 error: failed to run custom build command for `pgrx-pg-sys v0.11.4`
64.78 note: To improve backtraces for build dependencies, set the CARGO_PROFILE_RELEASE_BUILD_OVERRIDE_DEBUG=true environment variable to enable debug information generation.
64.78
64.78 Caused by:
64.78   process didn't exit successfully: `/root/workspace/target/16/debian_build/debian/postgresql-16-pgxman-pg-later-src/target/release/build/pgrx-pg-sys-5ed17adf970d8142/build-script-build` (exit status: 101)
64.78   --- stdout
64.78   cargo:rerun-if-env-changed=PGRX_BUILD_VERBOSE
64.78   cargo:rerun-if-env-changed=PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE
64.78   cargo:rerun-if-env-changed=PGRX_PG_CONFIG_PATH
64.78   cargo:rerun-if-env-changed=PGRX_PG_CONFIG_AS_ENV
64.78   cargo:rerun-if-env-changed=LLVM_CONFIG_PATH
64.78   cargo:rerun-if-env-changed=LIBCLANG_PATH
64.78   cargo:rerun-if-env-changed=LIBCLANG_STATIC_PATH
64.78   cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS
64.78   cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_aarch64-unknown-linux-gnu
64.78   cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS_aarch64_unknown_linux_gnu
64.78   cargo:rerun-if-env-changed=PGRX_PG_SYS_GENERATE_BINDINGS_FOR_RELEASE
64.78   cargo:rerun-if-changed=include
64.78   cargo:rerun-if-changed=cshim
64.78   cargo:rerun-if-changed=/root/workspace/target/16/debian_build/config.toml
64.78   cargo:rerun-if-env-changed=PGRX_TARGET_INFO_PATH_PG16_aarch64-unknown-linux-gnu
64.78   cargo:rerun-if-env-changed=PGRX_TARGET_INFO_PATH_PG16
64.78   cargo:rerun-if-env-changed=PGRX_BINDGEN_NO_DETECT_INCLUDES_aarch64-unknown-linux-gnu
64.78   cargo:rerun-if-env-changed=PGRX_BINDGEN_NO_DETECT_INCLUDES
64.78   cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER_PG16_aarch64-unknown-linux-gnu
64.78   cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER_PG16
64.78   cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER_aarch64-unknown-linux-gnu
64.78   cargo:rerun-if-env-changed=PGRX_INCLUDEDIR_SERVER
64.78
64.78   --- stderr
64.78   build_paths=BuildPaths { manifest_dir: "/root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.11.4", out_dir: "/root/workspace/target/16/debian_build/debian/postgresql-16-pgxman-pg-later-src/target/release/build/pgrx-pg-sys-f95eb0f1f1a09f9d/out", src_dir: "/root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.11.4/src/include", shim_src: "/root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.11.4/cshim", shim_dst: "/root/workspace/target/16/debian_build/debian/postgresql-16-pgxman-pg-later-src/target/release/build/pgrx-pg-sys-f95eb0f1f1a09f9d/out/cshim" }
64.78   Generating bindings for pg16
64.78   pg_config --configure CLANG = Some("/usr/bin/clang-17")
64.78   Bindgen found Ubuntu clang version 17.0.6 (9ubuntu1)
64.78   clang diag: /usr/include/postgresql/16/server/port.h:211:81: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:212:70: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:213:66: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:214:55: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:215:69: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:216:58: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:217:54: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/port.h:218:43: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/lib/stringinfo.h:96:67: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/lib/stringinfo.h:107:78: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:179:40: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:180:49: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:183:30: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:183:56: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:185:43: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:186:52: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:188:47: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:192:31: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:192:57: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:195:30: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:195:56: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:197:41: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:200:31: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:200:57: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:214:48: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:302:54: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/elog.h:537:47: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/palloc.h:162:44: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   clang diag: /usr/include/postgresql/16/server/utils/palloc.h:163:80: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
64.78   thread '<unnamed>' panicked at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bindgen-0.69.2/codegen/mod.rs:4206:17:
64.78   Invalid or unknown abi 16 for function "_ZGVnN4vv_atan2f" (Function { name: "_ZGVnN4vv_atan2f", mangled_name: Some("_ZGVnN4vv_atan2f"), link_name: None, signature: TypeId(ItemId(25257)), kind: Function, linkage: External })
64.78   stack backtrace:
64.78      0: rust_begin_unwind
64.78      1: core::panicking::panic_fmt
64.78      2: <bindgen::ir::function::Function as bindgen::codegen::CodeGenerator>::codegen
64.78      3: <bindgen::ir::item::Item as bindgen::codegen::CodeGenerator>::codegen
64.78      4: <bindgen::ir::module::Module as bindgen::codegen::CodeGenerator>::codegen::{{closure}}
64.78      5: <bindgen::ir::module::Module as bindgen::codegen::CodeGenerator>::codegen
64.78      6: <bindgen::ir::item::Item as bindgen::codegen::CodeGenerator>::codegen
64.78      7: bindgen::codegen::codegen::{{closure}}
64.78      8: bindgen::ir::context::BindgenContext::gen
64.78      9: bindgen::codegen::codegen
64.78     10: bindgen::Bindings::generate
64.78     11: bindgen::Builder::generate
64.78     12: build_script_build::run_bindgen
64.78     13: build_script_build::get_bindings
64.78     14: build_script_build::generate_bindings
64.78     15: build_script_build::main::{{closure}}::{{closure}}::{{closure}}
64.78   note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
64.78   thread 'main' panicked at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pgrx-pg-sys-0.11.4/build.rs:201:41:
64.78   thread panicked while generating bindings: Any { .. }
64.78   stack backtrace:
64.78      0: rust_begin_unwind
64.78      1: core::panicking::panic_fmt
64.78      2: core::result::unwrap_failed
64.78      3: core::result::Result<T,E>::expect
64.78      4: build_script_build::main::{{closure}}::{{closure}}
64.78      5: core::iter::adapters::map::map_try_fold::{{closure}}
64.78      6: <alloc::vec::into_iter::IntoIter<T,A> as core::iter::traits::iterator::Iterator>::try_fold
64.78      7: <core::iter::adapters::map::Map<I,F> as core::iter::traits::iterator::Iterator>::try_fold
64.78      8: <I as alloc::vec::in_place_collect::SpecInPlaceCollect<T,I>>::collect_in_place
64.78      9: alloc::vec::in_place_collect::from_iter_in_place
64.78     10: alloc::vec::in_place_collect::<impl alloc::vec::spec_from_iter::SpecFromIter<T,I> for alloc::vec::Vec<T>>::from_iter
64.78     11: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter
64.78     12: core::iter::traits::iterator::Iterator::collect
64.78     13: build_script_build::main::{{closure}}
64.78     14: std::thread::scoped::scope::{{closure}}
64.78     15: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
64.78     16: std::panicking::try::do_call
64.78     17: __rust_try
64.78     18: std::panicking::try
64.78     19: std::thread::scoped::scope
64.78     20: build_script_build::main
64.78     21: core::ops::function::FnOnce::call_once
64.78   note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
64.78 warning: build failed, waiting for other jobs to finish...
```

--- a/pgrx-bindgen/Cargo.toml
+++ b/pgrx-bindgen/Cargo.toml
@@ -16,7 +16,7 @@ proc-macro2.workspace = true
 syn.workspace = true
 walkdir.workspace = true
 
-bindgen = { version = "0.69", default-features = false, features = ["runtime"] }
+bindgen = { version = "0.69.4", default-features = false, features = ["runtime"] }
 clang-sys = { version = "1", features = ["clang_6_0", "runtime"] }
 quote = "1.0.33"
 shlex = "1.3" # shell lexing, also used by many of our deps

